logging/config/cluster-logging-configuring-cr.adoc

```diff
@@ -8,9 +8,32 @@ toc::[]
 
 To configure {logging-title} you customize the `ClusterLogging` custom resource (CR).
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
-
 include::modules/cluster-logging-about-crd.adoc[leveloffset=+1]
+
+////
+// collecting this information here for a future PR
+
+If you want to specify collector resources or deployments, you must create a `ClusterLogging` CR:
```

Comment:
..resources or scheduling..

Reply:
This won't be live content anyway, I've just added it in a comment so that we remember to include any new information when updating the clusterlogging docs.